test/apis/realtime/multi-container/build-cpu-tfs.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+# usage: build-cpu.sh [REGISTRY] [--skip-push]
+#   REGISTRY defaults to $CORTEX_DEV_DEFAULT_IMAGE_REGISTRY; e.g. 764403040460.dkr.ecr.us-west-2.amazonaws.com/cortexlabs or quay.io/cortexlabs-test
+
+image_name="realtime-multi-container-cpu-tfs"
```

Comment:
nit: I think I'd switch it so the file was named `build-tfs-cpu.sh` and the image was named `realtime-multi-container-tfs-cpu`, (also for web), what do you think?